sections/clause_08d_rest_api_create_update_delete.adoc

```diff
@@ -1,13 +1,52 @@
 === Modifying Data
 
 This section describes how to create, update and delete Entities.
-Deltails on how to send create, update and delete requests and how responses are received are described in the protocol bindings sections.
+Details on how to send create, update and delete requests and how responses are received are described in the protocol bindings sections.
 
 The content type of data sent to a service must be `application/json`, unless otherwise specified.
 
 ==== Creating Entities
 
 Entities can be created with a create request to an URI pattern that addresses an entity set (<<pattern_entityset>> or <<pattern_entityset_related>>).
+The create operation MUST follow the integrity constraints as defined in <<tab-integrity-constraint-create>>
```

Comment:
Integrity constraints are already in the data model. The API section should not contain any data model information.